data/human/deep jobs.txt

```diff
@@ -353,3 +353,617 @@ mission "Record academic conference"
+			`As you walk away from a food stand in the spaceport, a rather young man in activewear steps away from the wall and smoothly sidles up next to you, between one step and the next.`
+			`	"Hello Captain. Please don't be alarmed, do you have a moment?"`
+			`	His words come out very quickly as if they've been heavily rehearsed. Glancing around, so far as you can tell, no one appears to be paying any attention to either of you.`
+			choice
+				`	"Sure. Who are you?"`
+					goto who
+				`	"Sorry, but I'm busy right now."`
+			`	"Uh, I was coming to thank you if that helps?" he says.`
+			choice
+				`	"Oh. What is it?"`
+				`	"Sorry, maybe next time."`
+					decline
+			
+			label who
+			`	He continues walking and speaks without looking directly at you. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I'm uh, well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
+			choice
+				`	"Work is work."`
+					goto blindacceptance
+				`	"Ah, yes."`
+					goto blindacceptance
+				`	"Do all the secure packages belong to Deep Security?"`
+			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs.`
+			label blindacceptance
+			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
+			`	"So at any rate, I'm authorized to give you a one-time offer to join that list. Don't worry, I'm not asking anything of you right now."`
+			`	Belatedly, he seems to do a double-check that no one is actively listening to the conversation as you walk. "So, you interested?"`
+			choice
+				`	"Oh sure, I'm willing to help."`
+					goto agree
+				`	"Sorry, but I don't think I want to get that involved."`
+					decline
+				`	"You said 'at any rate' twice."`
+			`	He looks away sheepishly. "Er, yes, sorry about that. The question stands though."`
+			choice
+				`	"Oh sure, I'm willing to help."`
+					goto agree
+				`	"Sorry, but I don't think I want to get that involved."`
+					decline
+			label agree
+			`	"Fantastic," he says, quickly modulating his voice as someone briefly looks up at you from their phone. "I'll add you to the list as soon as I get back to the office again. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
+			apply
+				set "deep security reveal"
+			`	Agent Wayde flashes you a smile and jogs off ahead. Pushing his hair back out his eyes, he flicks a subtle salute your direction just before he turns a corner and disappears.`
```

Comment:
@Pointedstick it seems we're starting from very different baseline expectations so I sought specific feedback on the Discord. No one saw any specific issue with Julie so I have focused on a rework of Agent Wayde's interaction where I was able to get some useful insight. Hopefully this satisfies the bulk of your concerns?

```suggestion
			`As you turn to leave a food stand in the spaceport, a rather young man in activewear waves you over to a bench off to the side of the main drag.`
			choice
				`	(Go over to him.)`
					goto bench
				`	(Keep walking.)
			`	When you go to ignore him he gives a quick, sharp, whistle and briefly flashes a deep security badge at you. He immediately covers this with a friendly wave for any nearby eyes, but it is clear that he won't be ignored.`
			label bench
			`	When you approach the bench he says, "Hello Captain. Please, have a seat and allow me to take a moment of your time."`
			`	His manner is very rehearsed, but he seems quite intent and assured in his request.`
			choice
				`	"Sure. Who are you?" (Sit down.)`
					goto who
				`	"Sorry, but I'm busy right now."`
			`	"To be clear, I was here thank you if that helps?" he says.`
			choice
				`	"Oh. What is it?" (Sit down.)`
				`	"Sorry, maybe next time."`
					decline
			label who
			`	He smiles at you before directing his attention back to those passing by as he speaks. "My name is Wayde, Agent Wayde if you will. Don't mind the title though, I... well I work in the logistics division for Deep Security. You've transported a large number of secure packages recently and several of them were mine."`
			choice
				`	"Work is work."`
					goto blindacceptance
				`	"Ah, yes."`
					goto blindacceptance
				`	"Do all the secure packages belong to Deep Security?"`
			`	"Oh no, plenty of tech businesses and wealthy individuals use the same secure package shipping channels. We do monitor all of them though just to make sure our own channels don't get compromised by something ill-advised... or illegal." He shrugs, giving you a measured look.`
			label blindacceptance
			`	"Well, I looked at your record and it seems you've done quite a lot for the Deep, and you're well-regarded. We like to keep a list of pilots that we can call on if we need something delivered with either urgency or discretion, and it seems like you qualified. You passed the background check at any rate.`
			`	"So at any rate, I'm authorized to give you a one-time offer to join that list. Don't worry, I'm not asking anything of you right now."`
			`	He stops keeping an eye on passers-by for a moment to look directly at you. "So, you interested?"`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
				`	"You said 'at any rate' twice."`
			`	His expression twitches slightly but he carries on with a chuckle. "Er, good catch. The question stands though."`
			choice
				`	"Oh sure, I'm willing to help."`
					goto agree
				`	"Sorry, but I don't think I want to get that involved."`
					decline
			label agree
			`	"Excellent," he says, standing up and stretching as if concluding a friendly conversation. He turns back to you with a smile, "I'll add you to the list as soon as I get back to the office then. Your ID will also let you see which packages are ours specifically after that too. Oh and again, thank you!"`
			apply
				set "deep security reveal"
			`	Agent Wayde appears to flick a subtle salute your direction, or perhaps merely tips an imaginary cap. "Good day Captain, I look forward to working with you in the future."`
			`	With a quick motion he flicks his hair back out his eyes and jogs away, sending one last wave your direction just before he turns a corner and disappears.`
```

Reply:
That's better for sure. I would suggest a number of additional minor changes afterwards (for example, remove "I... well "), but nothing that yould dramatically change Wayde.